fluentui_persona/src/main/java/com/microsoft/fluentui/persona/AvatarView.kt

```diff
@@ -148,6 +147,20 @@ open class AvatarView : AppCompatImageView {
             invalidate()
         }
 
+    /**
+     * Defines the [ContentDescription] applied to the avatar.
+     */
+    var avatarContentDescriptionLabel: String = ""
+        set(value) {
+            if (field == value)
+                return
+
+            field = value
+            if(avatarContentDescriptionLabel.isNotEmpty()) {
```

Comment:
(same Q here ref should we be able to set it to empty)

Reply:
When empty, it announces as unlabelled something. So, thought not to allow that.

Reply:
Okay, thanks, I agree with you we should not want it to announce unlabeled when empty, we should definitely avoid that!